lib/BitrateTest.ts

```diff
@@ -502,11 +535,21 @@ export namespace BitrateTest {
      */
     errors: DiagnosticError[];
 
+    /**
+     * An array of ICE candidates gathered when connecting to media.
+     */
+    iceCandidates: RTCIceCandidateStats[];
```

Comment:
Hmm, odd to call this `iceCandidates` when it's an array of `iceCandidateStats`. Should we change one name or the other?

Reply:
Capturing our conversation, we will
```
Rename from
{
  iceCandidates: RTCIceCandidateStats[];
  selectedIceCandidatePair: {
    localCandidate: RTCIceCandidateStats;
    remoteCandidate: RTCIceCandidateStats;
  }
  // Other report properties
}
to
{
  iceCandidateStats: RTCIceCandidateStats[];
  selectedIceCandidatePairStats: {
    localCandidate: RTCIceCandidateStats;
    remoteCandidate: RTCIceCandidateStats;
  }
  // Other report properties
}
```